text/0014-soulbound-claims.md

```diff
@@ -0,0 +1,86 @@
+- FIP: 14
+- title: Soulbound tokens for identity claims
+- author: @molekilla (Rogelio Morrell)
+- status: draft
+- created: 2022-06-27
+
+# Summary
+
+Soulbound tokens for identity claims allows for a better user and developer experience by using existing Web3 technologies based on NFT token standards. In this FIP we described our findings and recommendations for FDP identity claims.
+
+# Context and motivation
+Decentralized Society is the next paradigm in the Ethereum ecosystem. Its focus is for `social use cases` where data is the main transferrable asset. A `Soul` represents an entity either a company, a group of people or a single person, which posses `Soulbound Tokens` or `SBTs`. These SBTs represent claims given to a Soul, these claims are `non-transferrable` and can be used as a `Verifiable Credential` or VC. 
+
+SBTs also allows for metadata to be either private (eg privacy-enabled with ZKP) or public. 
+
+
+# Guide-level explanation
+
+FDP claims can be upgraded to work with Soulbound tokens. This feature gives holders a more Web3 way, to verify provenance and trust. 
+
+- As an issuer, I need to grant claims for one or more assets / data items
+- As an issuer, I need to revoke claims
+- As a holder, I need to read my issued claims
+- As a holder, I need to share my issued claims with a verifier
+- As a verifier, I need to ask a holder for verification 
+- As a verifier, I need to verify holder proof / presentation
+
+# Reference-level explanation
+
+## Milestones
+
+## Milestone 1 - Issuer
+
+A Dapp with issuer capabilities can grant and revoke SBT to holders. The claims that the issuers creates are stored in Swarm, with the hash in the onchain SBT token.
+
+Depending on the use case, these claims can be revoked or renewed.
+
+The implementation must be able to:
+
+- Have SBT issuer smart contracts
+- JavaScripts / Typescript library

```

Comment:
I would link here the JS library for FDP contracts for reference: https://github.com/fairDataSociety/fdp-contracts/tree/master/js-library

Reply:
Regarding more details,  you are very right, that's the reason I wanted to get your feedback. Indeed, it might be technical bounty, but I need to speced it out